engines/python/setup/djl_python/input_parser.py

```diff
@@ -266,6 +266,11 @@ def parse_lmi_default_request_rolling_batch(payload):
             f"Invalid request payload. Request payload should be a json object specifying the 'inputs' field. Received payload {payload}"
         )
 
+    if not isinstance(inputs, str):
```

Comment:
Not sure about this case. Would empty str work?

Reply:
yea, empty str does succeed. Probably makes sense to validate it's nonempty. This is what llama2-7b returns for empty prompt
```
{"generated_text": "Tags: c#, asp.net, asp.net-mvc, asp.net-mvc-4, asp.net-mvc-5\n"}
```